langs/python.go

```diff
@@ -55,19 +55,19 @@ func (h *PythonLangHelper) Runtime() string {
 }
 
 func (h *PythonLangHelper) LangStrings() []string {
-	return []string{"python","python3.6"}
+	return []string{"python", "python3.6"}
 }
 
 func (h *PythonLangHelper) Extensions() []string {
 	return []string{".py"}
 }
 
 func (h *PythonLangHelper) BuildFromImage() (string, error) {
-	return fmt.Sprintf("python:%v", h.Version), nil
+	return fmt.Sprintf("python:%s-slim-stretch", h.Version), nil
 }
 
 func (h *PythonLangHelper) RunFromImage() (string, error) {
-	return fmt.Sprintf("fnproject/python:%v", h.Version), nil
+	return fmt.Sprintf("python:%s-slim-stretch", h.Version), nil
```

Comment:
I think this should stay being an image that we curate.

Reply:
There's no reason to do so, since `python:3.6.X-slim-stretch` is a public image from Docker library (not from someone's namespace) that needs exactly 2 packages.

BTW, this image is not used, because python doesn't use multi-stage builds, so the only image used - `python:3.6` which we do not control.

Reply:
Fine let's come back to the image thing  later - otheriwse this LGTM - good stuff!